components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.core/src/main/java/org/wso2/carbon/identity/configuration/mgt/core/dao/impl/CachedBackedConfigurationDAO.java

```diff
@@ -273,6 +275,22 @@ public boolean isExistingResource(int tenantId, String resourceId) throws Config
         return configurationDAO.isExistingResource(tenantId, resourceId);
     }
 
+    @Override
+    public void deleteResourcesByType(int tenantId, String resourceTypeId) throws ConfigurationManagementException {
+
+        List<Resource> resourceList = configurationDAO.getResourcesByType(tenantId, resourceTypeId);
+        if (resourceList.isEmpty()) {
+            if (log.isDebugEnabled()) {
+                log.debug("No resource found for the resourceTypeId: " + resourceTypeId);
```

Comment:
Add more content for this log. The tenant may be?

Reply:
Addressed with https://github.com/wso2/carbon-identity-framework/pull/5030/commits/89229d99304bf28cb30b59a43d181d33054a0be6